src/Components/ADT3DSceneBuilder/Internal/Behaviors/VisualRules/VisualRules.types.ts

```diff
@@ -1,5 +1,12 @@
-import { IExpressionRangeVisual } from '../../../../../Models/Types/Generated/3DScenesConfiguration-v1.0.0';
-
 export interface IVisualRulesListProps {
-    ruleItems: IExpressionRangeVisual[];
+    ruleItems: IVisualRule[];
+    onRemoveRule: (ruleId: string) => void;
+    onEditRule: (ruleId: string) => void;
+}
+export type IConditionsType = 'Badge' | 'Mesh coloring';
+
+export interface IVisualRule {
```

Comment:
don't we have a visual rule type @pasanchMSFT ? Can we share that here?

Reply:
These types we'll probably have to handle in a merge conflict, because it is something I overlooked when I did the schema change PR.

Reply:
imo, this can change in next PR.

Reply:
what do you mean? Do we not have an IVisualRule type in the app right now?